### PR TITLE
Combine jest and cypress reports

### PR DESCRIPTION
**Pull request by Joyfal**
## Checklist

- [N/A] Did you write tests for your code?
- [N/A] Did all the tests pass locally?

## Trello Ticket
https://trello.com/c/2P5wHequ/62-combining-jest-and-cypress-coverage-reports

## Short Description
See ticket above.

I also removed unnecessary jest test in deploy workflows, since we have a dedicated workflow for tests and coverage report that includes Jest and Cypress.

## Reviewer Notes

Anything special you want the reviewer to know?

### CODE REVIEW

File: .github/workflows/test_coverage.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Cypress E2E tests
+name: Jest and Cypress Test and Coverage
```
Comment (macedo22) [inline .github/workflows/test_coverage.yml:1]: This is nitpicky, but to reflect the change to this workflow, can we rename this file to something like `tests.yml`, `test.yml`, `test_and_coverage.yml`, or anything else that just captures that this workflow runs both our jest tests and cypress tests? The current name suggests it just runs our cypress tests. Let me know your thoughts!

**Review by macedo22 — COMMENTED** [review]
One small suggestion

File: .github/workflows/test_coverage.yml (hunk 1 above)
Comment (macedo22) [reply .github/workflows/test_coverage.yml:1]: Whoops, I didn't see that you already did this! I misread. I'll resolve this.

**Review by macedo22 — APPROVED** [review]

**Comment by macedo22** [comment]
Thanks for doing this! This is great

**Review by czhou29 — APPROVED** [review]

**Pull request merged**